queries.sql

```diff
@@ -15,3 +15,86 @@ SELECT * from animals WHERE neutered = TRUE;
 SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* TRANSACTIONS */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+
+/* update species */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+/* check deletion */ 
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+/* delete animals born after 2022 and update weight */
+
+BEGIN;
+
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
```

Comment:
- [ ] Before this query Add a query that `UPDATE` all animals' weight to be their weight multiplied by -1.
`UPDATE animals SET weight_kg = weight_kg * -1;`
then Rollback to the savepoint
`SAVEPOINT SP1;`

Reply:
I have done that [lines 59 - 62](https://github.com/leopiresgoss/vet-clinic-db/blob/9c7bdd8c011f52f96c892d11fbb978eaf1424bac/queries.sql#L59-L62)